src/utils/streams/__tests__/concat_stream.js

```diff
@@ -0,0 +1,79 @@
+import expect from 'expect.js';
+
+import {
+  createListStream,
+  createPromiseFromStreams,
+  createConcatStream
+} from '../';
+
+describe('concatStream', () => {
+  it('accepts an initial value', async () => {
+    const output = await createPromiseFromStreams([
+      createListStream([1,2,3]),
+      createConcatStream([])
+    ]);
+
+    expect(output).to.eql([1,2,3]);
```

Comment:
I meant `createConcatStream([0])` or something like that. Won't that end up as `[0,1,2,3]`? Might made the param a bit "clearer" in what role it has.